Library/Homebrew/cmd/test-bot.rb

```diff
@@ -1017,7 +1018,14 @@ def test_bot
       end
     end
   ensure
-    HOMEBREW_CACHE.children.each(&:rmtree) if ARGV.include? "--clean-cache"
+    if ARGV.include? "--clean-cache"
+      HOMEBREW_CACHE.children.each(&:rmtree)
+    else
+      Dir.glob("*.bottle*.tar.gz") do |bottle_file|
+        FileUtils.rm_f HOMEBREW_CACHE/bottle_file
+      end
+    end
```

Comment:
I don't think we want this to delete all bottles in this case? That will mean it redownloads bottles used as dependencies that haven't been updated every time. Feels nicer to store and delete the specific file.